src/types/index.ts

```diff
@@ -0,0 +1,12 @@
+export type firebaseConfigType = {
```

Comment:
In reference to [this comment](https://github.com/digidem/mapeo-webmaps/pull/68/files#r1048872408), we can just get rid of this file. And instead of exporting child type as an alias just use the `React.ReactNode` instead of `React.ReactElement`